homeassistant/components/zwave_me/light.py

```diff
@@ -51,7 +51,7 @@ def __init__(
         device: ZWaveMeData,
     ) -> None:
         """Initialize the device."""
-        super().__init__(self, controller=controller, device=device)
+        super().__init__(self, controller, device=device)
```

Comment:
Please be consistent with how the parameters are passed in accordance to what the signature of the target requires. All target parameters here accept positional arguments. So either pass positional arguments or use keywords. There's no reason to mix it up.